legend-depot-core-http/src/main/java/org/finos/legend/depot/core/http/BaseServer.java

```diff
@@ -108,7 +103,7 @@ public void run(T configuration, Environment environment)
             environment.jersey().setUrlPattern(urlPattern);
         }
         environment.jersey().register(MultiPartFeature.class);
-        environment.jersey().register(JacksonJaxbJsonProvider.class);
+        environment.jersey().register(BaseServerJacksonProvider.class);
```

Comment:
With this, we'll make sure to use a `JacksonJsonProvider` that is of our full control. But the gist here is `Jackson` supports auto-discovery for `JacksonJsonProvider` implementation and will use the object mapper configured in there. As such, we can create our `objectMapper` there and configure accordingly. The advantage of this approach over the traditional one (like in engine and SDLC` is that we can make use of the context resolver mechanism of `JacksonJsonProvider` to change object mapper for different Java classes, in particular, for this case, if we are returning `PureModelContextData`, we can use a specific object mapper (which, for example, configure `Include.NON_NULL` and sorts fields alphabetically), otherwise, we will use the standard object mapper which will not exclude `null` values.